File: JavaScript/M00_JavaScriptProgrammingBasics/L04_ForLoop/Lab/P06_VowelsSum.js
function characterSequence(input) {
    let text = input[0];

    let value = {
        a: 1,
        e: 2,
        i: 3,
        o: 4,
        u: 5
    }

    let sum = 0;
    for (let i = 0; i < text.length; i++) {
        if (value[text[i]]) {
            sum += value[text[i]];
        }
    }
    console.log(sum);
}

File: Python/M01_ProgrammingBasics/L04_ForLoop/Lab/Solutions/P06_VowelsSum.py
text = input()
sum = 0

for letter in text:
    if letter == "a":
        sum += 1
    elif letter == "e":
        sum += 2
    elif letter == "i":
        sum += 3
    elif letter == "o":
        sum += 4
    elif letter == "u":
        sum += 5
print(sum)